src/frontend/screens/MetaMaskHome/index.tsx

```diff
@@ -24,6 +24,7 @@ const MetaMaskHome = function ({ path = 'home.html' }: MetaMaskHomeProps) {
             ''
           )}`}
           allowpopups={trueAsStr}
+          key={'metamask_home' + path + hash}
```

Comment:
this isn't strictly necessary for this specific issue's fix, but a similar bug could occur so I'm going to go ahead and add the same fix